firmware/integration/rusefi_config.txt

```diff
@@ -1191,13 +1194,23 @@ uint8_t[4] unusuedsw;
 	uint8_t[IAC_PID_MULT_SIZE] iacPidMultLoadBins;;"Load",   1,   0.0,        0,  500.0,   2
 	uint8_t[IAC_PID_MULT_SIZE] iacPidMultRpmBins;;"RPM",        @@RPM_1_BYTE_PACKING_MULT@@,     0,      0.0,   12000.0,   0	
 
-	int[527] mainUnusedEnd;
+	int[479] mainUnusedEnd;
 
 ! end of engine_configuration_s
 end_struct
 
 engine_configuration_s engineConfiguration;
 
+
+float[AFTERSTART_HOLD_CURVE_SIZE] afterstartHoldTimeBins;;"C",        1,     0,   -100.0,    250.0,  0
+float[AFTERSTART_HOLD_CURVE_SIZE] afterstartHoldTime;;"Seconds",      1,     0,      0,    100,   1
+
+float[AFTERSTART_DECAY_CURVE_SIZE] afterstartDecayTimeBins;;"C",        1,     0,   -100.0,    250.0,  0
+float[AFTERSTART_DECAY_CURVE_SIZE] afterstartDecayTime;;"Seconds",      1,     0,      0,    100,   1
+
+float[AFTERSTART_ENRICH_CURVE_SIZE] afterstartEnrichBins;;"C",        1,     0,   -100.0,    250.0,  0
+float[AFTERSTART_ENRICH_CURVE_SIZE] afterstartEnrich;;"%",            100,     -1,      0,    600,   1
```

Comment:
I'm not sure the offset/scaling/limits are correct here.  This can also probably be a `uint8_t`

Reply:
Scaling are right I belive.
I have done 100 and -1 for most enrichments to get value in % in addition.

So that 0% are 0 enrichment

Reply:
Is that consistent with other fields in the firmware? Other multipliers are just that - a multiplier.  1.0 = no adjustment, 1.5 = add 50% extra.  Adding a setting for those is a separate problem, and whatever we add for now should be consistent with our other stuff.

Reply:
You are right, it should be consistent. This is how I have it, but doesnt match master.
Will change this.

Reply:
thanks!